File: C++/cold.cpp
#include <iostream>
using namespace std;

int main() {
  int n;
  cin >> n;

  int totalColdDays = 0;
  for (int i = 0; i < n; i++) {
    int temp;
    cin >> temp;

    if (temp < 0) {
      totalColdDays++;
    }
  }

  printf("%d\n", totalColdDays);
}

File: Java/cold.java
import java.io.*;
import java.util.*;

public class cold {

  public static void main(String[] args) throws IOException {
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    br.readLine();

    StringTokenizer st = new StringTokenizer(br.readLine());
    int totalColdDays = 0;

    while (st.hasMoreTokens()) {
      int temp = Integer.parseInt(st.nextToken());

      if (temp < 0) totalColdDays++;
    }

    System.out.println(totalColdDays);
  }
}